src/helics/apps/CMakeLists.txt

```diff
@@ -91,7 +91,9 @@ if(HELICS_BUILD_APP_LIBRARY)
                                                OR HELICS_DISABLE_ASIO)
     )
         target_compile_definitions(helics_apps PUBLIC BOOST_DATE_TIME_NO_LIB)
-        target_compile_definitions(helics_apps PUBLIC BOOST_BEAST_USE_STD_STRING_VIEW)
+        if(${Boost_VERSION_MINOR} LESS 81)
```

Comment:
What is Beast using in newer versions of Boost?

Reply:
In older versions, we need this flag set to use std::string_view.  In newer versions boost automatically picks the string_view and matches the types so the flag is useless and not needed.